UnityProject/Assets/Scripts/Players/PlayerManager.cs

```diff
@@ -67,8 +67,8 @@ public bool CreatePlayer(PlayerDTO playerDTO)
             activePlayers.Add(PlayerId(playerDTO.id), player);
 
             //Random Texture 
-            Material material = (Material)Resources.LoadAll("Materials/Players", typeof(Material))[UnityEngine.Random.Range(0, 100)];
-            player.GetComponent<Renderer>().material = material;
+            Material material = (Material)Resources.LoadAll("Materials/Players", typeof(Material))[UnityEngine.Random.Range(0, 4)];
```

Comment:
can we move this elsewhere please? (Olaf: we shouldn't reload all materials everytime we generate a player)